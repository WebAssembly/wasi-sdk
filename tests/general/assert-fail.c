#ifdef NDEBUG
#undef NDEBUG
#endif

#include <assert.h>
#include <stdbool.h>

int main(void) {
    assert(false);
    return 0;
}
