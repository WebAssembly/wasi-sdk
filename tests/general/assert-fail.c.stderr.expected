Assertion failed: false (assert-fail.c: main: 9)
Error: failed to run main module `assert-fail.c.---.wasm`

Caused by:
    0: failed to invoke ---
    1: error while executing at wasm backtrace:
       note: using the `WASMTIME_BACKTRACE_DETAILS=1` environment variable may show more debugging information
    2: wasm trap: wasm `unreachable` instruction executed
